code/modules/power/lighting.dm

```diff
@@ -674,7 +674,7 @@
 
 		to_chat(user, "You inject the solution into the [src].")
 
-		if(S.reagents.has_reagent("plasma", 5))
+		if(S.reagents.has_reagent("plasma", 5) || S.reagents.has_reagent("plasma_dust", 5))
 
 			log_admin("LOG: [user.name] ([user.ckey]) injected a light with plasma, rigging it to explode.")

```

Comment:
While you're in the file anyway, [user.name] ([user.ckey]) can be replaced with key_name(user) in log_admin and key_name_admin(user) in message_admins.